pymaker/dss.py

```diff
@@ -535,35 +511,59 @@ def validate_frob(self, ilk: Ilk, address: Address, dink: Wad, dart: Wad):
         assert rate != Ray(0)
         assert self.live()
 
-    def past_frobs(self, number_of_past_blocks: int, ilk=None) -> List[LogFrob]:
+    def past_frobs(self, from_block: int, to_block: int = None, ilk: Ilk = None, chunk_size=20000) -> List[LogFrob]:
```

Comment:
I always wondered what the reason was for `numer_of_past_blocks` in the (hopefully now) former interface.